diff --git a/node_modules/expo-modules-jsi/apple/Package.swift b/node_modules/expo-modules-jsi/apple/Package.swift
index 738d2f6..58a9193 100644
--- a/node_modules/expo-modules-jsi/apple/Package.swift
+++ b/node_modules/expo-modules-jsi/apple/Package.swift
@@ -1,4 +1,4 @@
-// swift-tools-version: 6.2
+// swift-tools-version: 6.0
 // The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import Foundation
